docs/user/builds.rst

```diff
@@ -1,8 +1,75 @@
-Build Process
+Build process
 =============
 
-Every documentation build has limited resources.
-Our current build limits are:
+Once a project has been imported into Read the Docs and a push is done (or a build is manually triggered),
+Read the Docs executes specific pre-defined jobs to build the project's documentation and update the hosted version with up-to-date content.
+This page explains in details what happens behind the scenes.
+
+
+Understanding what's going on
+-----------------------------
+
+Understanding how Read the Docs builds projects helps with debugging the problems that may appear in the process.
+It should also allow users to take advantage of certain things that happen during the build process.
+
+.. note::
+
+   All the steps are ran inside a Docker container with the image the project defines in :ref:`config-file/v2:build.os`.
+
+
+The following are the pre-defined jobs executed by default by Read the Docs:
+
+:checkout:
+
+   Checks out project's code from the URL's repository defined for this project.
+   It will use `git clone`, `hg clone`, etc depending on the VCS backend.
+
+:system_dependencies:
+
+   Installs OS/system dependencies.
+   This includes specific version of languages (e.g. Python, NodeJS, Go, Rust) and also OS packages.
+
+   At this point, :ref:`config-file/v2:build.tools` can be used to defined languages versions,
+   and :ref:`config-file/v2:build.apt_packages` to define APT packages.
+
+:create_environment:
+
+   Creates a Python environment to install all the dependencies in an isolated and reproducible way.
+   Depending on what's defined by the project a Virtualenv or a Conda environment (:ref:`config-file/v2:conda`) will be used.
```

Comment:
```suggestion
   Depending on what's defined by the project a virtualenv or a conda environment (:ref:`config-file/v2:conda`) will be used.
```

Do we not have a reference for virtualenv?

Reply:
I'll take a look, but I don't remember right now. I think we don't because it's the default and there is nothing to set up when using virtualenv.